### DISCUSSION TITLE
How can I use useHover() with an <input type="date">?

### DISCUSSION THREAD

**Discussion by osdiab** [floating-ui/floating-ui#1806] [Q&A] [answered]
This came up at my company - with React DOM Interactions, if I use an `input` of type `date`, and then you try to interact with the date picker (in Chrome), then it will open but as soon as the mouse enters the date picker UI, the floating UI will be dismissed.

Anyone know how one would work with this? Thanks! [Here's a repro CodeSandbox](https://codesandbox.io/s/aged-meadow-zyrxv4).

**Comment by atomiks** [comment, marked as answer]
Here's one solution (this also uncovered a bug which is commented): https://codesandbox.io/s/immutable-tdd-pd9xif

It does close if the cursor leaves the datepicker, not sure if that's desirable

**Reply by osdiab** [reply]
thanks for this! in my case i think the UX would be better if it didn't so i guess we could just keep it disabled/switch it to a click trigger once it opens or something.

**Reply by osdiab** [reply]
nm can't figure out how to do that since i don't understand what is actually triggering the popover to close when you mouse out.

**Reply by osdiab** [reply]
nm again, i just updated floating ui in the sandbox to the latest version, and it works exactly as expected! https://codesandbox.io/s/pensive-herschel-yvzdmn?file=/src/Popover.tsx
